tools/testharness.py

```diff
@@ -1,547 +1,535 @@
 #!/usr/bin/env python3
 
+from argparse import ArgumentParser
+from itertools import chain
+from os import environ, sched_getaffinity
+from pathlib import Path
+import re
+import subprocess
 import sys
-import os
-import os.path
-import glob
-import time
-from io import StringIO
+from textwrap import dedent, indent
+from time import monotonic, sleep
+from xml.etree.ElementTree import parse
+# Warning: The xml.etree.ElementTree module is not secure against maliciously
+# constructed data. If you need to parse untrusted or unauthenticated data see
+# XML vulnerabilities.
+# Use defusedxml instead?
+# from defusedxml.ElementTree import parse
```

Comment:
I don't think this is an issue - we're only parsing committed .xml files - and people can in fact already execute whatever they like from these .xml files.